pkg/reconciler/instances/istio/action.go

```diff
@@ -97,6 +98,10 @@ func (a *MainReconcileAction) Run(context *service.ActionContext) error {
 
 		err = performer.Update(context.KubeClient.Kubeconfig(), istioManifest.Manifest, istioStatus.TargetVersion, context.Logger)
 		if err != nil {
+			errPatchMutatingWebhook := performer.PatchMutatingWebhook(context.Context, context.KubeClient, context.Logger)
+			if errPatchMutatingWebhook != nil {
+				context.Logger.Info("Could not patch MutatingWebhookConfiguration")
```

Comment:
In case error appears we should propagate it to the upper layer. Even if we decided to log it then it should be `Error` instead of `Info`